tower-http/Cargo.toml

```diff
@@ -83,7 +83,7 @@ auth = ["base64"]
 catch-panic = ["tracing", "futures-util/std"]
 cors = []
 follow-redirect = ["iri-string", "tower/util"]
-fs = ["tokio/fs", "tokio-util/io", "tokio/io-util", "mime_guess", "mime", "percent-encoding", "httpdate", "set-status", "futures-util/alloc"]
+fs = ["tokio/fs", "tokio-util/io", "tokio/io-util", "mime_guess", "percent-encoding", "httpdate", "set-status", "futures-util/alloc"]
```

Comment:
Actually you should add a new `validate-request` feature that adds the `mime` dependency, like we do with the other middleware. That way users only have to pull down `mime` if they're actually using it.

Reply:
Added, thanks